pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -193,6 +194,12 @@ func metricsTimerTask(ctx *zedagentContext, handleChannel chan interface{}) {
 			ctx.ps.CheckMaxTimeTopic(wdName, "publishMetrics", start,
 				warningTime, errorTime)
 
```

Comment:
Makes sense to call 		ctx.ps.StillRunning(wdName, warningTime, errorTime)
here since we are doing a second send attempt to LOC.

Reply:
Done.